README.md

```diff
@@ -35,6 +35,7 @@ See [Maven Repository](http://central.maven.org/maven2/org/codelibs/gitbucket/gi
 
```

Comment:
Would you mind updating the link to the Maven Repository as well?
`gitbucket-fess-plugin_2.12` -> `gitbucket-fess-plugin_2.13`

Reply:
done!